kolibri/plugins/device_management/assets/src/views/manage-content-page/channel-list-item-large.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+
+  <k-grid cols="12">
+    <k-grid-item size="2">
+      <slot name="thumbnail"></slot>
+    </k-grid-item>
+    <k-grid-item size="8">
+      <k-grid cols="8">
```

Comment:
In order to implement items that span multiple rows, it's necessary to nest grids. Note here that the child grid has its number of columns set to the size of the parent grid item.

This works here because the parent grid is _not_ responsive - it is a template that's _only_ used at large window sizes.